src/panels/config/dashboard/ha-config-navigation.ts

```diff
@@ -77,6 +84,16 @@ class HaConfigNavigation extends LitElement {
 
   private _entryClicked(ev) {
     ev.currentTarget.blur();
+    if (
+      ev.currentTarget.parentElement.href.endsWith(
+        "#external-app-configuration"
+      )
```

Comment:
Not sure I like this, the url will stay changed, and it adds a entry to the history...

Reply:
This is how it used to be, but in `ha-sidebar`